spec/requests/linking/link_defendant_maat_reference_v2_spec.rb

```diff
@@ -37,7 +37,7 @@
     let(:maat_invalid_reference) do
       {
         title: 'Unable to link the defendant using the MAAT ID.',
-        message: 'If this problem persists, please contact the IT Helpdesk on 0800 9175148.'
+        message: 'The MAAT reference you provided is not available to be associated with this defendant.'
```

Comment:
We could clarify the message.
Do we mean to say that the maat does not exist?
we could just say: "The MAAT reference you provided does not exist"
or
"The MAAT reference you provided does not exist and therefore cannot be associated with this defendant"